lib/webvalve/manager.rb

```diff
@@ -70,6 +70,7 @@ def reset
       allowlisted_urls.clear
       fake_service_configs.clear
       stubbed_urls.clear
+      WebMock.reset!
```

Comment:
This was required to prevent global state from mucking up the test run and making this test flake:
https://github.com/Betterment/webvalve/pull/41/files#diff-4f60a691d33035e7b6c85487ba4937efR55

I was surprised that we leave existing WebMock stubs around even when we clear out the local registry of stubbed URLs. But maybe this shouldn't go here? I could put it in that one test's `before` hook instead? 😅

Reply:
seems fine. i think reset is only used internally.